### ISSUE TITLE
68 landmarks and 900 faces on the world largest selfie

### ISSUE THREAD

**Issue by edumucelli** [deepinsight/insightface#836]
Hi @nttstar, thanks for the great work and for the effort of sharing it here. I have three questions.

1. I can't reproduce the ~900 faces identified on the largest selfie image on the paper (RetinaFace: Single-stage Dense Face Localisation in the Wild). Using the threshold of `0.5` as indicated in the paper's figure label gave me 388 faces as shown below:

From the paper 900 faces
![900](https://user-images.githubusercontent.com/959623/62493283-fe5b7d00-b7d0-11e9-8532-d6288cc0482c.png)

From the code on this repository using threshold of `0.5` and `net3` "only" 388 faces
![388](https://user-images.githubusercontent.com/959623/62493193-cf450b80-b7d0-11e9-8f04-b8aa78568474.jpg)

2. What is the trade-off of the `treshold` itself, how (de) increasing it will change the capacity of the detection?

3. On the paper you have compared a 68-landmark model. Are you going to publish pre-trained model with 68 landmarks?

Thanks in advance.

**Comment by szad670401** [comment]
Try to test with multi-scale

**Comment by njho** [comment]
Is the 68 point LM available?
